app/models/user.rb

```diff
@@ -8,7 +8,7 @@ def validate(record)
 
 class User < ActiveRecord::Base
   self.table_name = 'rusers'
-  attr_accessible :username, :email, :password, :password_confirmation, :openid_identifier, :key, :photo, :photo_file_name
+  attr_accessible :username, :email, :password, :password_confirmation, :openid_identifier, :key, :photo, :photo_file_name, :location_privacy
```

Comment:
Is there a reason to have location privacy be set in a unique column on user, rather than on the location table? Could you imagine someone wanting one location private and another not, both associated with them in some way? Perhaps this belongs on the location tag table.